src/docs/asciidoc/wan/tuning.adoc

```diff
@@ -423,7 +423,7 @@ the aforementioned elements is shown below.
 Finally, as we've mentioned, the exact values which will give you the optimal performance depend on your environment and use case.
 Please benchmark and try out different values to find out the right values for you.
 
-==== Discovery period (Discovery SPI)
+==== Discovery Period (Discovery SPI)
```

Comment:
I'm not sure the discovery related settings should be under the `Tuning WAN Replication` section. The `Using the Discovery SPI` section under `Defining WAN Replication` is long and these topics seem to be related to each other. WDYT about extracting these under an `Endpoint Discovery` section or similar?

Reply:
The reason that they are here is their advanced nature when configuring the Discovery SPI. The aim is like giving the basics in the beginning of the chapter, and then dive into the details later on. But still, let me think about this.